ui/packages/catalog-ui-search/src/main/webapp/component/editor/editor.view.js

```diff
@@ -47,10 +47,14 @@ const template = require('./editor.hbs')
 const CustomElements = require('../../js/CustomElements.js')
 const DetailsFilterView = require('../dropdown/details-filter/dropdown.details-filter.view.js')
 const DropdownModel = require('../dropdown/dropdown.js')
-const DetailsInteractionsView = require('../details-interactions/details-interactions.view.js')
 const PopoutView = require('../dropdown/popout/dropdown.popout.view.js')
```

Comment:
build failed because `PopoutView` and _ are no longer used

Reply:
removed 👍